mne_bids/write.py

```diff
@@ -2543,18 +2551,24 @@ def mark_channels(bids_path, *, ch_names, status, descriptions=None, verbose=Non
             f"({len(ch_names)})."
         )
 
-    if not all(status in ["good", "bad"] for status in status):
+    if not all(status_ in ["good", "bad"] for status_ in status):
         raise ValueError(
             'Setting the status of a channel must only be "good", or "bad".'
         )
 
     # Read sidecar and create required columns if they do not exist.
     if "status" not in tsv_data:
-        logger.info('No "status" column found in input file. Creating.')
+        logger.info(
+            'No "status" column found in input file.'
+            'Creating it with default value "good".'
```

Comment:
(here and below) it seems odd to call it "input file" since the user never passed the `*_channels.tsv` filename to the function (only the BIDSPath)

```suggestion
            'No "status" column found in channels file.'
            'Creating it with default value "good".'
```